recipe/meta.yaml

```diff
@@ -33,8 +33,7 @@ requirements:
     - jinja2 >=3.0.3
     - jupyter_core
     - jupyter-lsp >=1.5.1
-    - jupyter_server >=1.16.0,<3
-    - jupyter_ydoc >=0.2.0,<0.3.0
+    - jupyter_server >=2.0.0rc3,<3
```

Comment:
Might be blocking on `jupyter_server` 2 to be available on conda.

Reply:
- https://github.com/conda-forge/jupyter_server-feedstock/pull/102
  - https://github.com/conda-forge/staged-recipes/pull/21200

Reply:
this will need to add the `jupyter_server_rc` channel:
https://github.com/conda-forge/jupyter_server-feedstock/pull/102/files#diff-999d2f034982a9a29cc0f5a5340497f76ff9243576ea74e5954ec5ad5f32b74cR6

Reply:
https://anaconda.org/conda-forge/jupyter_server/files?channel=jupyter_server_rc